app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,19 @@
 <resources>
     <string name="app_name">SpeakOut</string>
+    <string name="content_description_app_logo">App logo.</string>
+
+    <!--  Login Activity  -->
+    <string name="login_success_activity_login">Login Succeeded.</string>
```

Comment:
Split the strings in regions, kind of a good practice to became easy to search for something.